src/DynamoCore/Graph/Annotations/AnnotationModel.cs

```diff
@@ -66,6 +66,8 @@ public override double Width
             }
             set
             {
+                if (width == value) return;
```

Comment:
`{return; }`

Reply:
as long as the statement is on the same line as the condition we dont need `{}` according to Dynamos coding standards